Adyen/Service/Checkout.cs

```diff
@@ -293,8 +292,7 @@ public CreateCheckoutSessionResponse Sessions(CreateCheckoutSessionRequest creat
         public async Task<CreateCheckoutSessionResponse> SessionsAsync(CreateCheckoutSessionRequest createCheckoutSessionRequest)
         {
             var jsonRequest = createCheckoutSessionRequest.ToJson();
-            var jsonResponse = await _sessions.RequestAsync(jsonRequest);
```

Comment:
Why not update the entire class? That way we could also maybe remove the dependency for this from this class

Reply:
Indeed it is only in one place :(

Reply:
Ah yeah I just wanted to test it and thought we'd change the rest later during the service automation. I can change it manually as well though!

Reply:
Changed it :)